dashboard/config/blocks/GamelabJr/gamelab_setDefaultSpriteSize.json

```diff
@@ -0,0 +1,30 @@
+{
+  "category": "Actions",
+  "config": {
+    "func": "setDefaultSpriteSize",
+    "blockText": "set new sprite default size to {SIZE}",
+    "args": [
+      {
+        "name": "SIZE",
+        "options": [
+          [
+            "large",
```

Comment:
[out of curiosity] do you now how strings like this are translated?

Reply:
I don't know the details, but all the visible text on blocks is pulled into the pipeline and translated automatically. (as in, no need to add it to one of our i18n strings files, just being in the block config is sufficient)